inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -86,8 +83,57 @@ void MKLDNNSplitNode::initSupportedPrimitiveDescriptors() {
         THROW_IE_EXCEPTION << "The sizes of input blob and sum of output blobs are not equal.";
     supportedPrimitiveDescriptors.emplace_back(config, impl_desc_type::ref, outFormats);
 
-    auto numOfDim = static_cast<size_t>(srcDims.ndims());
+    //Support per channel format
+    outFormats.clear();
+    auto perChannelFormat = MKLDNNMemory::GetPerChannelFormat(srcDims);
+    config.inConfs[0].desc = MKLDNNMemoryDesc(srcDims, inputDataType, perChannelFormat);
+    for (size_t i = 0; i < outDims.size(); ++i) {
+        auto o_Dims = outDims[i];
+        config.outConfs[i].desc = MKLDNNMemoryDesc(o_Dims, outputDataType, perChannelFormat);
+        outFormats.push_back(perChannelFormat);
+    }
+    supportedPrimitiveDescriptors.emplace_back(config, impl_desc_type::ref, outFormats);
```

Comment:
I believe, that we use emplace_back for performance reason, but do we have enough space in supportedPrimitiveDescriptors for that? Can be reserve(possible_desc_num) called at the beginning of function to avoid of  full relocation?

Reply:
Done c1d47d09c49b216880a8b0b07848d33e7bb88372.